java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -956,42 +956,30 @@ public static Table readJSON(Schema schema, JSONOptions opts, HostMemoryBuffer b
         opts.isDayFirst(), opts.isLines()));
   }
 
-  /**
-   * Read a Parquet file using the default ParquetOptions.
-   * @param path the local file to read.
-   * @return the file parsed as a table on the GPU.
-   */
-  public static Table readParquet(File path) {
-    return readParquet(ParquetOptions.DEFAULT, path);
-  }
-
   /**
    * Read a Parquet file.
    * @param opts various parquet parsing options.
    * @param path the local file to read.
    * @return the file parsed as a table on the GPU.
    */
   public static Table readParquet(ParquetOptions opts, File path) {
+    if (opts.getIncludeColumnNames().length == 0) {
+      throw new IllegalArgumentException("Column names must be provided");
+    }
     return new Table(readParquet(opts.getIncludeColumnNames(),
         path.getAbsolutePath(), 0, 0, opts.timeUnit().typeId.getNativeId()));
   }
 
-  /**
-   * Read parquet formatted data.
-   * @param buffer raw parquet formatted bytes.
-   * @return the data parsed as a table on the GPU.
-   */
-  public static Table readParquet(byte[] buffer) {
-    return readParquet(ParquetOptions.DEFAULT, buffer, 0, buffer.length);
-  }
-
   /**
    * Read parquet formatted data.
    * @param opts various parquet parsing options.
    * @param buffer raw parquet formatted bytes.
    * @return the data parsed as a table on the GPU.
    */
   public static Table readParquet(ParquetOptions opts, byte[] buffer) {
+    if (opts.getIncludeColumnNames().length == 0) {
```

Comment:
This looks like an undesirable change and is reflected in the corresponding update to the tests.  If the caller must specify all columns to read from the file, how does the caller read a Parquet file when they don't know the columns in the file?  (e.g.: doing dynamic schema discovery). This used to be possible with the Java API but now seems like it isn't.